project/src/store/reducer.js

```diff
@@ -71,6 +72,27 @@ const reducer = (state = initialState, action) => {
         nearby: action.payload,
       };
     }
+    case ActionType.USER_EMAIL: {
+      return {
+        ...state,
+        userEmail: action.payload,
+      };
+    }
+    case ActionType.REVIEW_LIST: {
+      return {
+        ...state,
+        reviews: action.payload,
+        //allComment: state.reviews,
+      };
+    }
+    case ActionType.NEW_COMMENTS: {
+      return {
+        ...state,
+        comment: action.payload,
```

Comment:
Тебе тут нужно соединить со всеми старыми просто

```suggestion
        comments: state.comments.concat(action.payload),
```